packages/miniflare/src/workers/core/proxy.worker.ts

```diff
@@ -69,7 +69,15 @@ function isPlainObject(value: unknown) {
 	);
 }
 function objectContainsFunctions(obj: Record<string, unknown>): boolean {
```

Comment:
This should remove the need for @ts-ignore below

```suggestion
function objectContainsFunctions(obj: Record<string | symbol, unknown>): boolean {
```

Reply:
Fixed in https://github.com/cloudflare/workers-sdk/pull/6301/commits/a454995bb087b5fb647801e1d9a42d8c3dba6df5

Reply:
🤦👍